core/config/project_settings.cpp

```diff
@@ -1258,6 +1258,8 @@ ProjectSettings::ProjectSettings() {
 	GLOBAL_DEF("application/config/custom_user_dir_name", "");
 	GLOBAL_DEF("application/config/project_settings_override", "");
 
+	GLOBAL_DEF_RST("application/config/text_to_speech", true);
```

Comment:
I wonder if `application/config/` is the best location for this. Currently it's used for very general project config (name, main scene, user dir, etc.).

Might be better under `audio/driver/` or `audio/general/`?

Reply:
Moved to `audio/general/text_to_speech`.